testgrid/conformance/creating-a-bucket.md

```diff
@@ -0,0 +1,21 @@
+# Creating Federated Conformance Test GCS Buckets
+
+This guide is aimed primarily at members of the Google GKE EngProd team for 
+creating Google provided GCS buckets to be used by other providers for hosting
+conformance results on TestGrid, but the general steps should be good practice
+for anyone setting up a GCS bucket for auotomated uploads.
```

Comment:
typo error: s/auotomated/automated/

Reply:
Thanks for these. seems our spellcheck presubmit is not up to snuff 🙃

Reply:
fixed